### ISSUE TITLE
About the papaer of RobustScanner

### ISSUE THREAD

**Issue by Gaoxj2020** [open-mmlab/mmocr#507]
Did you used all the mjsynth training sets to get the results in the paper ? I see that there are two types of MJ dataset annotation files , shuffle_labels.txt and label.txt. Which annotation file did you use to get the result got by only using on dataset mj+st in papers.

**Comment by gaotongxiao** [comment]
Please see the notes here https://mmocr.readthedocs.io/en/latest/datasets/recog.html#mjsynth-syn90k
